apps/api/src/app/controllers/category.controller.ts

```diff
@@ -18,9 +18,6 @@ export class CategoryController {
         const invariantValidationErrors = result.filter(isInternalError);
 
         if (invariantValidationErrors.length > 0) {
-            console.log({
-                invalidDTOS: invariantValidationErrors.map((e) => e.toString()),
-            });
             return res
                 .status(httpStatusCodes.internalError)
```

Comment:
I've seen this repeated a lot. Can we
- create a helper for this
- investigate what we need above Nest's built in behaviour 
- refine error reporting through api
- consider creating custom middleware to simplify the code